manifests/config.pp

```diff
@@ -30,4 +30,6 @@
     mode   => '0755',
   }
 
+  $services_real = prefix($katello::services_list, $katello::services_directory)
+  ensure_resource('file', $services_real, {'ensure' => 'file', 'mode' => '0644'})
```

Comment:
In that case ensure_resource is dependant on the order (unless both use ensure_resource). Maybe a step back: why are these files needed even though they don't have any content (here)? In case you do want content, wouldn't a define be better? Similar to how we have ```foreman::plugin``` and then many ```foreman::plugin::*``` classes.